data/data/ovirt/variables-ovirt.tf

```diff
@@ -0,0 +1,36 @@
+variable "bootstrap_dns" {
+  type        = string
+  default     = true
+  description = "Whether to include DNS entries for the bootstrap node or not."
+}
+
+variable "ovirt_url" {
+  type        = string
+  description = "The oVirt engine URL"
+}
+
+variable "ovirt_username" {
+  type        = string
+  description = "The name of user to access oVirt engine API"
+}
+
+variable "ovirt_password" {
+  type        = string
+  description = "The plain password of user to access oVirt engine API"
+}
+
+variable "ovirt_cafile" {
+  type        = string
+  default     = ""
```

Comment:
does `ovirt_cafile` need this default?

Reply:
will remove it, I don't think it is in use